solr/webapp/web/libs/angular-selectize.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Angular Selectize2
+ * https://github.com/machineboy2045/angular-selectize
```

Comment:
Should reference the license in this file? https://github.com/machineboy2045/angular-selectize/blob/master/LICENSE

Reply:
I am backing out this javascript library in favour of `chosen.js` one.